### PR TITLE
Add DA report commands

### PR DESCRIPTION
**Pull request by jomrazek**
### Checklist:

* [ ] Have you added a note in the [CHANGELOG wiki](https://github.com/project-ncl/bacon/wiki/Changelog) for your change if user-facing?
* [ ] Have you added unit tests for your change?

### CODE REVIEW

File: da/src/main/java/org/jboss/bacon/da/DAReportsCli.java [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+                ObjectHelper.print(false, result);
+            } catch (IOException e) {
+                throw new RuntimeException(e);
+            }
+            return 0;
+        }
+    }
+
+    @CommandLine.Command(
+            name = "lookup-versions-npm",
```
Comment (jomrazek) [inline da/src/main/java/org/jboss/bacon/da/DAReportsCli.java:183]: @janinko we wanted to keep lookup for npm versions right? this one is not in new lookup endpoint

Comment (janinko) [reply da/src/main/java/org/jboss/bacon/da/DAReportsCli.java:183]: yup

**Comment by thescouser89** [comment]
Our code is getting sexy with WLStripper! LGTM! THanks for the PR!

**Review by thescouser89 — APPROVED** [review]
LGTM

**Pull request merged**